neptune/new/exceptions.py

```diff
@@ -885,3 +885,29 @@ def __init__(self, location: str):
 class ArtifactUploadingError(NeptuneException):
     def __init__(self, msg: str):
         super().__init__("Cannot upload artifact: {}".format(msg))
+
+
+class NeptuneUnsupportedArtifactFunctionalityException(NeptuneException):
+    def __init__(self, functionality_info: str):
+        message = """
+{h1}
+----NeptuneUnsupportedArtifactFunctionality-------------------------------------
+{end}
+It seems you are using Neptune Artifacts functionality "{functionality_info}" that is currently not supported.
+
+{correct}Need help?{end}-> https://docs.neptune.ai/getting-started/getting-help
+"""
+        super().__init__(message.format(functionality_info=functionality_info, **STYLES))
+
+
+class NeptuneEmptyLocationException(NeptuneException):
+    def __init__(self, location: str):
+        message = """
+{h1}
+----NeptuneEmptyLocationException----------------------------------------------
+{end}
+Neptune could not find files in the requested location ({location}).
```

Comment:
This exception will be raised asynchronously so without any context user can have some problem to guess what is it about. Let's be more verbose and friendly.